public/static/logkeeper.css

```diff
@@ -37,8 +37,7 @@ td.log pre {
   border-right:1px solid #ccc;
   cursor:pointer;
 }
-.global{
-  font-style:italic;
+.global{ // placeholder for global styles
```

Comment:
Use `/* */` for CSS comments. https://www.w3.org/TR/css-syntax-3/#comment-diagram

Reply:
Done